container-tools/confd-deb-sdk/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Generic ConfD client application builder
+#
+# Start from the debian linux base image.
+FROM debian:9.8
```

Comment:
As noted in another comment: Use Debian 10 "Buster" instead, as ConfD 7.2 support libcrypto 1.1 if you rebuild crypto